spyder/plugins/editor/widgets/gotoline.py

```diff
@@ -62,8 +62,10 @@ def __init__(self, editor):
 
         ok_button = bbox.button(QDialogButtonBox.Ok)
         ok_button.setEnabled(False)
+        # QIntValidator does not handle '+' sign
+        # See Spyder PR #20070
```

Comment:
Lets change this to:
```suggestion
        # See spyder-ide/spyder#20070
```
With that format you can ctrl+click and open a browser fro the link if the file was openned with Spyder